src/StatusQ/Components/private/SkeletonAnimation.qml

```diff
@@ -0,0 +1,29 @@
+import QtQuick 2.14
```

Comment:
just wondering if it makes sense to put this under `StatusQ/Animations` and start building our animations from there?